src/renderer/app.vue

```
@@ -34,6 +35,15 @@
     ApiKeyModal,
   },
   setup() {
+    const { t } = useI18n();
+
+    const convCauseToErrorMessage = (cause: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)